chart/tests/deployment_test.yaml

```diff
@@ -342,3 +342,35 @@ tests:
     - equal:
         path: spec.template.spec.containers[0].readinessProbe.periodSeconds
         value: 60
+- it: should set startupProbe periodSeconds to 60
+  set:
+    startupProbe.periodSeconds: 60
+  asserts:
+    - equal:
+        path: spec.template.spec.containers[0].startupProbe.periodSeconds
+        value: 60
+    - equal:
+        path: spec.template.spec.containers[0].startupProbe.failureThreshold
+        value: 1
+    - equal:
+        path: spec.template.spec.containers[0].startupProbe.httpGet.port
+        value: 80
+    - equal:
+        path: spec.template.spec.containers[0].startupProbe.httpGet.path
+        value: /healthz
+- it: should set startupProbe failureThreshold to 10
+  set:
+    startupProbe.failureThreshold: 10
+  asserts:
+    - equal:
+        path: spec.template.spec.containers[0].startupProbe.periodSeconds
+        value: 30
+    - equal:
+        path: spec.template.spec.containers[0].startupProbe.failureThreshold
+        value: 10
+    - equal:
+        path: spec.template.spec.containers[0].startupProbe.httpGet.port
+        value: 80
+    - equal:
+        path: spec.template.spec.containers[0].startupProbe.httpGet.path
+        value: /healthz
```

Comment:
Nit: You could add a "no startup probe exists if you don't specify something" test as well